core/src/main/scala/chisel3/internal/Builder.scala

```diff
@@ -210,15 +210,24 @@ private[chisel3] trait HasId extends InstanceId {
   // Uses a namespace to convert suggestion into a true name
   // Will not do any naming if the reference already assigned.
   // (e.g. tried to suggest a name to part of a Record)
-  private[chisel3] def forceName(prefix: Option[String], default: => String, namespace: Namespace): Unit =
+  private[chisel3] def forceName(
+    prefix:     Option[String],
+    default:    => String,
+    namespace:  Namespace,
+    errorIfDup: Boolean = false,
+    refBuilder: String => Arg = (x: String) => (Ref(x))
```

Comment:
```suggestion
    refBuilder: String => Arg = Ref(_)
```
or `x => ref`
Just FYI, since you have the type already defined you can use short-hand on functions like this.